### ISSUE TITLE
MenuHelper: just check for page editor?

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1333]
@blilreed63 

I'm still getting exceptions in MenuHelper when doing things in the theme editor. Earlier fixes checked menu.properties, but now sometimes I get 'menu' is undefined.

Can we just put a check in MenuHelper to see if the current editor is a page editor so that MenuHelper doesn't try to do things when the theme editor is running?

**Comment by peller** [comment]
that might just be masking a bug.

Which revision are you running and how do you reproduce the problem?

**Comment by peller** [comment]
in particular, are you running with 86d9685d817f49f8fedef30dc13ab87e420fb8cd

**Comment by JonFerraiolo** [comment]
yes, I pulled down that commit and I'm seeing a different exception occasionally in the theme editor where "menu" is undefined. The commit checks to see if menu.properties is undefined, not menu.

But isn't the menu helper only supposed to act on things in the page editor? Why would we want this logic to run at all when in the theme editor?

**Comment by peller** [comment]
Aren't all helpers generally active in the theme editor, just as no-ops?

**Comment by JonFerraiolo** [comment]
If a helper gets invoked by the theme editor (which is case now), then the helper is a no-op only if the helper does nothing. We could head off at pass by having calling function check to see if current editor is page editor.

**Comment by peller** [comment]
And this helper should do nothing in the theme editor case.  We could kludge around this by avoiding the bug when in theme editor.

But as an aside, do we use helper functionality at all in the theme editor?

**Comment by jhpedemonte** [comment]
-> M6

**Comment by peller** [comment]
-> M7

**Comment by peller** [comment]
-> M8
